airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py

```diff
@@ -54,6 +54,10 @@ def assert_emitted_at_increase_on_subsequent_runs(self, first_read_records, seco
     def assert_two_sequential_reads_produce_same_or_subset_records(
         self, records_1, records_2, configured_catalog, ignored_fields, detailed_logger
     ):
+        # should we re-use how we compare records at this point or just completely remove this assertion? Implications:
```

Comment:
I'm fine with killing this

Reply:
It is killed. I'll organize the funerals